EffectivePotential/include/effectivepotential/one_loop_potential.hpp

```diff
@@ -50,10 +50,10 @@ class OneLoopPotential : public Potential {
   /** Finite-temperature effective potential */
   double V(Eigen::VectorXd phi, double T) const;
   /** Zero-temperature one-loop correction */
-  double V1(std::vector<double> scalar_masses_sq,
-            std::vector<double> fermion_masses_sq,
-            std::vector<double> vector_masses_sq) const;
-  double V1(Eigen::VectorXd phi) const;
+  virtual double V1(std::vector<double> scalar_masses_sq,
```

Comment:
Why are these now virtual?

Reply:
In on-shell like scheme, only the  expression of V1 is different. So I think making it virtual may be usefully for some situations.